### PR TITLE
WW-90 update communitypage-admin-welcome-message & communitypage-policy-module-link-page-name

### PR DESCRIPTION
**Pull request by dianafa**
https://wikia-inc.atlassian.net/browse/WW-90

### CODE REVIEW

File: extensions/wikia/CommunityPage/CommunityPage.i18n.php [hunk 1]
```diff
@@ -267,7 +267,7 @@
 	'communitypage-deadendpages-title' => 'Seiten ohne Links',
 	'communitypage-deadendpages-description' => 'Diese Seiten sollten andere Artikel in deinem Wiki verlinken. Gut verlinkte Artikel können deinen Leser noch mehr Informationen zu deinem Thema vermitteln und außerdem dein Suchmaschinenranking verbessern.',
 	'communitypage-view-more' => 'Mehr anzeigen',
-	'communitypage-policy-module-link-page-name' => 'Hilfe:Community-Richtlinien',
```
Comment (sebastianmarzjan) [inline extensions/wikia/CommunityPage/CommunityPage.i18n.php:270]: This does not look like an intended change (same below)

File: extensions/wikia/CommunityPage/CommunityPage.i18n.php [hunk 2]
```diff
@@ -5,7 +5,7 @@
 	'communitypage-title' => 'Community',
 	'communitypage-header-welcome' => 'Welcome to the {{SITENAME}} community!',
 	'communitypage-header-join-button' => 'Join Now',
-	'communitypage-admin-welcome-message' => 'Welcome to {{SITENAME}}. Help us grow this wiki by editing and adding content. We welcome and appreciate your contributions! Check back here often for the latest updates and information about this community.',
+	'communitypage-admin-welcome-message' => 'Share your knowledge by editing pages, uploading images, and adding new content.',
```
Comment (sebastianmarzjan) [inline extensions/wikia/CommunityPage/CommunityPage.i18n.php]: nitpick: is the comma after images correct?

**Comment by sebastianmarzjan** [comment]
LGTM

**Comment by dianafa** [comment]
LGTM

**Pull request merged**